### PR TITLE
Fix typo in Layout section of persistent list

### PR DESCRIPTION
**Pull request by mlodato517**
**This PR**
Just fixes a small typo at the end of the "Layout" section for the persistent linked list.

### CODE REVIEW

File: src/third-layout.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ warning: field is never used: `next`
    |     ^^^^^^^^^^^^^
 ```
 
-Seems legit. Rust continues to be a *completely* trivial to write. I bet we can just
+Seems legit. Rust continues to be *completely* trivial to write. I bet we can just
```
Comment (mlodato517) [inline src/third-layout.md:130]: Or, perhaps,
```suggestion
Seems legit. Rust continues to be a *completely* trivial language to write. I bet we can just
```

**Comment by Gankra** [comment]
The sentence could perhaps be rewritten to be easier to read, but it's currently more idiomatic as-is. ("today I wrote some Rust" is perfectly normal to say).

**Comment by mlodato517** [comment]
I’m a little confused by your comment 😅 I definitely agree that “today I wrote some Rust" is a valid sentence. How does that relate to the current “Rust continues to be a completely trivial to write.”? That still doesn’t sound like a correct sentence to me. I feel like the “a” is lost in space.

I don’t want to waste your time and understand this can’t be high priority, but could you sub out the word “Rust” or give some other example that makes sense here?

**Comment by Gankra** [comment]
Oh! Geez sorry, I got confused by your "perhaps" comment and missed that it wasn't the proposed change!

**Review by Gankra — COMMENTED** [review]
thanks!

**Review by Gankra — APPROVED** [review]

**Pull request merged**